public/components/common/query_utils/index.ts

```diff
@@ -61,14 +62,24 @@ export const convertDateTime = (
   datetime: string,
   isStart = true,
   formatted = true,
-  isMetrics: boolean = false
+  isMetrics: boolean = false,
+  isOtel: boolean = false
```

Comment:
i thought otel is also a metric, do we need two booleans? why not use subType

Reply:
subType/metricType variable isn't available in the functions that calls `convertDateTime` function, also there are two different ways for formatting date for otel and prometheus metric. if we use one boolean, it would either true or false but in our case `convertDateTime` function is used by wide range of functions which don't need metric date formatting. If its kept as single boolean then either of the metric formatting would be applied to all date time conversion even if not a metric query. Hence a single boolean is not possible here

Reply:
maybe i can create a subType, try and pass that. I'll try it that way

Reply:
changed it, will update now